Sources/Purchasing/NonSubscriptionTransaction.swift

```diff
@@ -40,6 +40,9 @@ public final class NonSubscriptionTransaction: NSObject {
     /// Paid price for the subscription
     @objc public let price: ProductPaidPrice?
 
+    /// Whether or not the purchase was made in sandbox mode.
+    @objc public let isSandbox: Bool
```

Comment:
Just to make sure, this won't be optional for non-subscription transactions right? Just making sure there won't be any issues parsing this.

Reply:
Also, might be good to add to some test.

Reply:
I checked the https://www.revenuecat.com/docs/api-v1#tag/customers/operation/subscribers and it's there. Also, wasn't optional for my tests, but worth double-checking

Reply:
For reference, we've checked the backend and it's there so we're good.